packages/transport/src/modules/sync/externalEventProcessors/PeerDeletedEventProcessor.ts

```diff
@@ -0,0 +1,30 @@
+import { Serializable, serialize, validate } from "@js-soft/ts-serval";
+import { CoreDate, CoreId } from "../../../core";
+import { PeerDeletedEvent } from "../../../events";
+import { PeerDeletionInfo, PeerDeletionStatus } from "../../relationships/local/PeerDeletionInfo";
+import { Relationship } from "../../relationships/local/Relationship";
+import { BackboneExternalEvent } from "../backbone/BackboneExternalEvent";
+import { ExternalEventProcessor } from "./ExternalEventProcessor";
+
+class PeerDeletedEventData extends Serializable {
+    @serialize()
+    @validate()
+    public relationshipId: string;
+
+    @serialize()
+    @validate()
+    public completedAt: string;
```

Comment:
Why don't we generally use the Core-types here?

Reply:
we do :)

Reply:
ah, no. we don't. This comes from the backbone and we only use it to validate the backbone payloads